core/client.go

```diff
@@ -19,6 +19,7 @@ type Client struct {
 	NetworkSpeedProfiles            *NetworkSpeedProfilesClient
 	Networks                        *NetworksClient
 	Organizations                   *OrganizationsClient
+	SecurityGroups                  *SecurityGroupsClient
```

Comment:
A new `SecurityGroupsClient` instance needs to also be assigned to the `Client` isntance in the `New()` function.